src/app/components/Embeds/Uploader/index.tsx

```diff
@@ -4,11 +4,11 @@ import path from 'ramda/src/path';
 import pathOr from 'ramda/src/pathOr';
 import { jsx } from '@emotion/react';
 import { ServiceContext } from '../../../contexts/ServiceContext';
+import { LeftChevron, RightChevron } from '../../icons';
+import CallToActionLink from '../../CallToActionLink';
```

Comment:
We should probably list the components we need to regression test as part of this refactor - shall we add something to either the JIRA ticket or the PR description?

Reply:
since we'll be refactoring the `liteSiteCTA` afterwards, for now there are no changes to any other components.

this change was caused by the imports being reordered

Reply:
Ah my apologies. Makes sense. Do we have a ticket for the refactor? If not, can we create one, and add a link to this thread so we don't forget?

Reply:
yeah i'll write up a ticket, probably by this Friday. and link this PR and thread

Reply:
ticket has been made now https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-543